src/data/queries/administration.ts

```diff
@@ -0,0 +1,18 @@
+import { FieldAdministration } from '@/types/drupal/field_type'
+import { Administration } from '@/types/formatted/administration'
+
+/**
+ * Formats a Drupal administration field into our standardized Administration interface.
+ *
+ * In our Drupal content, the `field_administration` field points to a taxonomy Term
+ * that is currently called "Section", but "Section" is only a human label added to
+ * it in the Drupal interface, and that name has evolved over time. It was originally,
+ * "administration" (and thus the name of the field). Since the only machine name of
+ * it is `field_administration`, we'll continue to call it "administration" in our code.
+ */
+export const formatAdministration = (
```

Comment:
I don't know if I care yet, but this _does_ deviate from the pattern. In `src/data/queries/*.ts`, we export `formatter()` and then `import { formatter as formatAdministration } from '...'` rather than defining it in here as `formatAdministration`.

For consistency, we should probably keep it the same, but I'm not strongly opinionated about this.

Reply:
72183573